### PR TITLE
[Integrations] Testcontainers, Kubernetes Operator, Kratix

### LINKED ISSUE #4025: OSS projects that integrate with Dapr
There are a number of other OSS project that have integrate with Dapr that need to be called out in the Dapr docs. 
This is to add a new topic in this section https://docs.dapr.io/developing-applications/integrations/ called "OSS Projects" that contains a list of other projects, notable

- Dapr K8s Operator https://operatorhub.io/operator/dapr-kubernetes-operator
- Dapr Kratix Marketplace https://github.com/syntasso/kratix-marketplace/tree/main/dapr
- Dapr Test Containers Module https://github.com/diagridio/testcontainers-dapr

### PR DESCRIPTION
**Pull request by hhunter-ms**
## Description

Add 'landing pages' for Dapr Testcontainers module, Dapr Kubernetes Operator, and the Dapr promise in Kratix Marketplace to the Integrations section of docs

## Issue reference

PR will close: #4025

### CODE REVIEW

File: daprdocs/content/en/developing-applications/integrations/Diagrid/test-containers.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+type: docs
+title: "How to: Integrate using Diagrid's Testcontainers Dapr Module"
+linkTitle: "Diagrid Testcontainers"
+weight: 3000
+description: "Use the Dapr Testcontainer module from your Java application"
+---
+
+You can use the Testcontainers Dapr Module provided by Diagrid to set up Dapr locally in your Java applications. Simply add the following dependency to your Mave project:
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/integrations/Diagrid/test-containers.md]:
```suggestion
You can use the Testcontainers Dapr Module provided by Diagrid to set up Dapr locally for your Java applications. Simply add the following dependency to your Maven project:
```

File: daprdocs/content/en/developing-applications/integrations/kratix-marketplace.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+description: "Integrate with Kratix using a Dapr promise"
+---
+
+As part of the [Kratix Marketplace](https://docs.kratix.io/marketplace), Dapr can be used to build custom platforms tailored to your needs. 
+
+{{% alert title="Note" color="warning" %}}
+The Dapr Helm chart generates static public and private key pairs that are published in the repository. This promise should only be used _locally_ for demo purposes. If you wish to use this promise for more than demo purposes, it's recommended to manually update all the secrets in the promise with keys with your own credentials.
+{{% /alert %}}
+
+Get started by simply installing the Dapr Promise, which installs DApr on all matching clusters.
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/integrations/kratix-marketplace.md]:
```suggestion
Get started by simply installing the Dapr Promise, which installs Dapr on all matching clusters. 
```

File: daprdocs/content/en/developing-applications/integrations/Diagrid/test-containers.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+type: docs
+title: "How to: Integrate using Diagrid's Testcontainers Dapr Module"
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/integrations/Diagrid/test-containers.md]:
```suggestion
title: "How to: Integrate using Testcontainers Dapr Module"
```

File: daprdocs/content/en/developing-applications/integrations/Diagrid/test-containers.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+linkTitle: "Diagrid Testcontainers"
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/integrations/Diagrid/test-containers.md]:
```suggestion
linkTitle: "Dapr Testcontainers"
```

File: daprdocs/content/en/developing-applications/integrations/kubernetes-operator.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+title: "How to: Use the Dapr Kubernetes Operator"
+linkTitle: "Dapr Kubernetes Operator"
+weight: 7000
+description: "Use the Dapr Kubernetes Operator to manage the Dapr lifecycle"
+---
+
+The Dapr Kubernetes Operator manages the lifecycle for Dapr and its components. Use the operator to automate the tasks required when oeprating Dapr in Kubernetes mode. 
+
+{{% alert title="Note" color="primary" %}}
+The Dapr Kubernetes Operator is currently a Dapr sandbox project.
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/integrations/kubernetes-operator.md]: You can remove this comment. It should be in dapr org as per https://github.com/dapr/community/issues/425

File: daprdocs/content/en/developing-applications/integrations/kubernetes-operator.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+{{% /alert %}}
+ 
+{{< button text="Install and use the Dapr Kubernetes Operator" link="https://github.com/dapr-sandbox/dapr-kubernetes-operator" >}}
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/integrations/kubernetes-operator.md]:
```suggestion
{{< button text="Install and use the Dapr Kubernetes Operator" link="https://github.com/dapr/dapr-kubernetes-operator" >}}
```

**Review by msfussell — COMMENTED** [review]
@salaboy - Please review this PR also

File: daprdocs/content/en/developing-applications/integrations/kubernetes-operator.md [hunk 7]
```diff
@@ -0,0 +1,11 @@
+---
+type: docs
+title: "How to: Use the Dapr Kubernetes Operator"
+linkTitle: "Dapr Kubernetes Operator"
+weight: 7000
+description: "Use the Dapr Kubernetes Operator to manage the Dapr lifecycle"
+---
+
+The Dapr Kubernetes Operator manages the lifecycle for Dapr and its components. Use the operator to automate the tasks required when oeprating Dapr in Kubernetes mode.
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/integrations/kubernetes-operator.md]:
```suggestion
The Dapr Kubernetes Operator manages the lifecycle for Dapr and its components. Use the operator to automate the tasks required when operating Dapr in Kubernetes mode. 
```

File: daprdocs/content/en/developing-applications/integrations/kubernetes-operator.md [hunk 8]
```diff
@@ -0,0 +1,11 @@
+---
+type: docs
+title: "How to: Use the Dapr Kubernetes Operator"
+linkTitle: "Dapr Kubernetes Operator"
+weight: 7000
+description: "Use the Dapr Kubernetes Operator to manage the Dapr lifecycle"
+---
+
+The Dapr Kubernetes Operator manages the lifecycle for Dapr and its components. Use the operator to automate the tasks required when operating Dapr in Kubernetes mode.
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/integrations/kubernetes-operator.md]:
```suggestion
You can use the Dapr Kubernetes Operator to manage the Dapr control plane. Use the operator to automate the tasks required to manage the lifecycle of Dapr control plane in Kubernetes mode. 
```

File: daprdocs/content/en/developing-applications/integrations/kubernetes-operator.md [hunk 9]
```diff
@@ -0,0 +1,11 @@
+---
+type: docs
+title: "How to: Use the Dapr Kubernetes Operator"
+linkTitle: "Dapr Kubernetes Operator"
+weight: 7000
+description: "Use the Dapr Kubernetes Operator to manage the Dapr lifecycle"
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/integrations/kubernetes-operator.md]:
```suggestion
description: "Use the Dapr Kubernetes Operator to manage the Dapr control plane"
```

**Review by msfussell — APPROVED** [review]
LGTM

**Comment by salaboy** [comment]
LGTM too.. this looks like a good starting point, can we rebase and merge?

**Pull request merged**